addons/mail/static/src/xml/many2one_avatar_user.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <!-- MailMany2OneAvatar: do not display the display_name in kanban views -->
+    <!-- MailMany2OneAvatar: do not display the display_name in kanban views unless requested -->
     <t t-name="mail.KanbanMany2OneAvatarUser" t-extend="web.Many2OneAvatar">
         <t t-jquery="img" t-operation="attributes">
             <attribute name="t-att-title">value</attribute>
         </t>
-        <t t-jquery="span" t-operation="replace"/>
+        <t t-jquery="span[t-if*='widget.mode']" t-operation="replace">
```

Comment:
Mmmh.

How about:
```xml
<t t-jquery="span[t-if*='widget.mode']" t-operation="attributes">
    <atttribute name="t-if">widget.mode === 'readonly' &amp;&amp; widget.displayAvatarName</attribute>
</t>
```

Seems a bit better since the base implementation already displays "value" in a t-esc so you can just change the condition.

Reply:
You are WRONG !

...

"attribute" takes two 2 then 1, not 3 then 1.